src/main/java/simplifier/controllers/LinkController.java

```diff
@@ -0,0 +1,31 @@
+package simplifier.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import simplifier.model.Link;
+import simplifier.services.LinkService;
+
+@RestController
+@RequestMapping("/link")
+public class LinkController {
+
+    private LinkService linkService;
+
+    @Autowired
+    public void setLinkService(LinkService linkService) {
+        this.linkService = linkService;
+    }
+
+    @PostMapping
+    public ResponseEntity<?> createLink(@RequestBody Link link) {
```

Comment:
Make dto object without fields that should not be saved, e.g. id and clicks. API should be easy to use by clients. Change incoming json to this structure
{
	"originalLink": "https://github.com/matsiukvadzim/URL-Simplifier/pull/10/files",
	"shortenedLink": "short",
	"description": "PR",
	"tags": [
               "quality",
               "test"
        ]
	"author":"putin"
}